docs/src_snippets/howto/python_api/c_applying_methods.py

```diff
@@ -23,7 +23,7 @@ def apply_class_method_clumping(summary_stats: SummaryStatistics) -> StudyLocus:
     from gentropy.method.window_based_clumping import WindowBasedClumping
 
     clumped_summary_statistics = WindowBasedClumping.clump(
-        summary_stats, window_length=500_000
```

Comment:
@addramir are we on board with this window size? All data processing uses the default value, changing this would mean changing the distances between the semi indices in the next release.